src/dataflow/lib/iot.ts

```diff
@@ -10,6 +10,7 @@ const AWS_IOT_ENDPOINT_HOST = "a2zxjwmcl3eyqd-ats.iot.us-east-1.amazonaws.com";
 // WTD This is for testing purposes only. Remove when we add users and hub owners.
 const OWNER_ID = "123";
 const SEND_INTERVAL = 1;
+const MAX_HUBS = 200;
```

Comment:
We can make this even bigger.  Maybe 1000 is a better starting point?  Thoughts?

Reply:
No-one is going to scroll through a list of 1000 hubs - for now, I would go with this and we'll have to work on a more efficient way to group devices on the control panel in future.